partner_location_auto_create/README.rst

```diff
@@ -0,0 +1,71 @@
+Partner Location Auto Create
+============================
+
+This module allows to create locations automatically when creating a partner.
+One use case of this feature is to allow a company not only to manage the stocks
+in its own warehouses, but also the warehouses of its suppliers.
+For example, an hatchery manages farms. Each farm is a partner and has its own inventory.
+The hatchery doesn't own the farms but manages their stocks.
+
+Whether the partner is a customer, a supplier or both, the proper locations will be created and
+the fields property_stock_customer and property_stock_supplier will be filled automatically.
+
+The modules also adds a button on the partner form to view the related locations.
+
+Installation
+============
+
+To install this module, you just need to select the module and ensure yourself dependencies are available.
+
+Configuration
+=============
+
+You must enable product.group_purchase_pricelist or product.group_sale_pricelist to use this feature.
+This is required to view the stock location properties on the partner view.
+
+In the company form, select the default partner and customer locations. This will set the default parent
+location for the partner locations.
+
```

Comment:
Please replace with:

```
* Add users to the group "Sales Pricelists" or "Purchase Pricelists" to view the stock location properties on the partner view.
* In the company form, select the default customer and supplier locations. This will set the default parent locations for the partner locations.
```